homeassistant/components/homematic/battery_sensor.py

```diff
@@ -0,0 +1,49 @@
+"""Support for HomeMatic binary sensors."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.const import STATE_UNKNOWN
+from . import ATTR_DISCOVER_DEVICES, HMDevice
+
+ATTR_LOW_BAT = "LOW_BAT"
+ATTR_LOWBAT = "LOWBAT"
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ["homematic"]
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
`battery_sensor` is not an entity component that exists in HA, how does this work ?